internal/service/alertconfiguration/model_test.go

```diff
@@ -561,3 +561,79 @@ func TestAlertConfigurationSdkToDSModelList(t *testing.T) {
 		})
 	}
 }
+
+func TestIntervalMinValue(t *testing.T) {
+	five := int64(5)
+	ten := int64(10)
+	zero := int64(0)
+	three := int64(3)
+
+	testCases := map[string]struct {
+		intervalMin    *int64
+		expectedResult *int
+		typeName       string
+	}{
+		"GROUP notification with nil interval_min defaults to 5": {
+			typeName:       "GROUP",
+			intervalMin:    nil,
+			expectedResult: admin.PtrInt(5),
+		},
+		"GROUP notification with interval_min=0 defaults to 5": {
+			typeName:       "GROUP",
+			intervalMin:    &zero,
+			expectedResult: admin.PtrInt(5),
+		},
+		"GROUP notification with interval_min=3 (less than min) defaults to 5": {
+			typeName:       "GROUP",
+			intervalMin:    &three,
+			expectedResult: admin.PtrInt(5),
+		},
```

Comment:
If users defines a value smaller than 5 what is the result? I suspect an `inconsistent error result after apply` as config value != state value.